app/app.go

```diff
@@ -144,11 +145,10 @@ func (a *App) Handle404(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *App) getSystemInstallDate() (int64, *model.AppError) {
-	result := <-a.Srv.Store.System().GetByName(model.SYSTEM_INSTALLATION_DATE_KEY)
-	if result.Err != nil {
-		return 0, result.Err
+	systemData, appErr := a.Srv.Store.System().GetByName(model.SYSTEM_INSTALLATION_DATE_KEY)
```

Comment:
No, because we need to use `err` later in the function to handle an `error` type, so I have to use different variables and is a good way to differentiate them, one is an `error` and the other is a `model.AppError`, so `err` and `appErr` makes sense.